src/sphinx/installation.rst

```diff
@@ -222,3 +222,93 @@ but instead
 
  package leon.test.myTestPackage
 
+
+Using Leon in IntelliJ
+----------------------
+
+You can maybe encounter some problems installing Leon, because of the various dependencies needed.
+Here is a step-by-step guide having as a goal to help you to install your first Leon project and to be sure that every
+dependency is present and correctly linked.
+
+The advantage of using IntelliJ IDEA is that it provides a Scala plugin which allows you to directly import SBT project
+into it.
+
+**Installation:**
+
+* `SBT 0.13.x <http://www.scala-sbt.org/>`_ (as described above)
+* `IntelliJ IDEA <https://www.jetbrains.com/idea/download/>`_
+* Leon project files, available in GitHub (as described above)
+* Java SE Development Kit 8 or 7 (as described above)
+* Support for at least one external solver (See :ref:`smt-solvers`)
+
+**Setup STB in your project:**
+
+Leon requires the Build class, which is a class created by SBT in its folder *target* when you compile your project the
+first time. Also, it will be comfortable to run Leon through terminal command, in order to easily specify arguments.
+We will resolve both of theses things running the command:
+
+.. code-block:: bash
+
+ $ sbt clean compile
+ $ sbt package script
+
+This will create the *leon* executable file that you can now use typing ```./leon```. You are not required to recompile
+this file after modifying your project, because *leon* file is actually a script that modifies the
+scala compiler, giving it the path to Leon, and then compile the file you specified. You can open *leon*
+in terminal ```cat leon``` to realize that.
```

Comment:
to realize means to create in English. Perhaps you can remove the last sentence, because its is no way documenting.